includes/emails/class-wc-email-customer-on-hold-order.php

```diff
@@ -40,6 +40,7 @@ public function __construct() {
 
 			// Triggers for this email.
 			add_action( 'woocommerce_order_status_pending_to_on-hold_notification', array( $this, 'trigger' ), 10, 2 );
+			add_action( 'woocommerce_order_status_processing_to_on-hold_notification', array( $this, 'trigger' ), 10, 2 );
```

Comment:
Customer should already receive an email saying `Your {site_title} order has been received!`, this email suppose to register that the order was on "processing" status, but got demoted to "on-hold" for some reason.